src/oid4vci/mod.rs

```diff
@@ -365,47 +366,33 @@ pub async fn oid4vci_exchange_credential(
                 JwtVcJson(response) => {
                     log::trace!("processing a JwtVcJson");
                     let rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
You may be able to remove this runtime instance and specify the runtime in the export macro, i.e.:

```rust
#[uniffi::export(async_runtime = "tokio")]
pub async fn oid4vci_exchange_credential ...
```

Reply:
This requires the `tokio` feature in uniffi, which was added when #43 was merged.

```toml
uniffi = { version = "0.28.1", features = ["cli", "tokio"] }
```